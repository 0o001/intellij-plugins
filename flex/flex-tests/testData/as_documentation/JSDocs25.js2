/**
* test: shown
*/
class foo
{
}
var x:fo<caret>o