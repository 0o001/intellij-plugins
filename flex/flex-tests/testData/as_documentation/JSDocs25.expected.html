<div class='definition'><pre>foo</pre></div><div class='content'><p> test: shown</div><table class='sections'></table>